.github/workflows/docker-unified.yml

```diff
@@ -437,6 +445,14 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: "3.7"
+      - name: Pull images in background
+        run: |
+          docker pull acryldata/datahub-actions:head &
+          docker pull confluentinc/cp-kafka:7.2.0 &
```

Comment:
Question: How are you thinking of keeping these images dependencies up to date?

Reply:
Manually currently as part of PR review process.

Reply:
Can you add a note where in the docker files to also update here in case of any changes?